packages/mathtype-ckeditor5/src/integration.js

```diff
@@ -254,6 +254,9 @@ export default class CKEditor5Integration extends IntegrationModel {
             );
         }
 
+        // Fire the 'change' event.
+        // this.editorObject.fire('change');
+
```

Comment:
Do we leave this?

Reply:
We added this callback to the version 4 of the plugin.
It may be a good a idea to do the same for the next version 
of the plugin. Otherwise, users who would upgrade and are
using this feature would miss it. 

But we decided to comment it so it'd need to be properly tested.

Reply:
The integration for CK5 is vastly different. I believe we should leave this change out, or at least add a comment explaining why it's there, so as to avoid any confusion when we see this piece of code in the future and might think that it used to be uncommented.